mintapi/signIn.py

```diff
@@ -283,11 +279,28 @@ def get_stable_chrome_driver(download_directory=os.getcwd()):
         )
 
     zip_file = zipfile.ZipFile(io.BytesIO(request.content))
-    zip_file.extractall(path=download_directory)
```

Comment:
What problem are we solving by rewriting this function?

Reply:
@snowskeleton Thanks for the feedback! 

The issue we're solving here is that the new chromedriver download zip now have a subdirectory (i.e. `chromedriver-win64`) instead of the chromedriver binary and license files. So this new function extracts the files from the subdirectory to the download directory whereas the `extractall` function just extracts the subdirectory to the download directory causing some filepath issues.

Old chromdriver download with no subdirectory:
https://chromedriver.storage.googleapis.com/114.0.5735.90/chromedriver_win32.zip

New chromedriver downoad with subdirectory:
https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.96/win32/chromedriver-win32.zip